EIPS/eip-7723.md

```diff
@@ -57,6 +57,12 @@ Non-Core EIPs that are `Considered for Inclusion` **SHOULD** be supported prior
 
 An EIP **MAY** be moved from `Considered for Inclusion` to `Declined for Inclusion` if client teams are against including the EIP in the network upgrade. 
 
+An EIP **SHOULD** have a Python implementation in [execution-specs](https://github.com/ethereum/execution-specs) in an open PR.
```

Comment:
One thing our team discussed was changing this to any client implementation, out of concern for overhead. A neutral (and arguably disposable) implementation of an EIP prior to it being considered is a high bar, and not strictly necessary to fill tests against. Our thoughts are nuanced on this matter though.

This doc could also provide some guidance on when it is ok for an exception to the SHOULD, and set the expectation that ACD rough consensus would be the arbiter of that exception.

Reply:
Appreciate the feedback, we could add a line regarding the CFI'd stage requirements on the basis of an ACD consensus.
 
Regarding the EELS overhead, the really nice thing about having everything in EELS from the start is that when the time comes to join all EIPs together to fill tests for the entire fork, everything is one implementation instead of scattered on different client implementations, because it's a big bottleneck to testing to fill one part of the tests with one t8n tool and then the rest with another.